common/djangoapps/student/tests/factories.py

```diff
@@ -226,3 +241,84 @@ class Meta:
     user = None
     secondary_email = factory.Sequence('robot+test+recovery+{}@edx.org'.format)
     is_active = True
+
+
+class BetaTesterFactory(UserFactory):
```

Comment:
These are used all over the place and contain no LMS-specific code, so lets put them somewhere everyone can/should access.